### PR TITLE
Fix github_oauth_token in the mobile world

### PR DESCRIPTION
**Pull request by MihaiTabara**

### CODE REVIEW

File: docker.d/scriptworker.yml [hunk 1]
```diff
@@ -28,5 +28,4 @@ task_log_dir: { "$eval": "TASK_LOGS_DIR" }
 watch_log_file: true
 scriptworker_provisioners:
     - { "$eval": "PROVISIONER_ID" }
-    - scriptworker-prov-v1
```
Comment (MihaiTabara) [inline docker.d/scriptworker.yml:31]: This is cleanup, we no longer need this.

File: pushapkscript/docker.d/init_worker.sh [hunk 2]
```diff
@@ -96,6 +96,8 @@ case $COT_PRODUCT in
         test_var_set 'GOOGLE_CREDENTIALS_FOCUS'
         test_var_set 'GOOGLE_PLAY_SERVICE_ACCOUNT_REFERENCE_BROWSER'
         test_var_set 'GOOGLE_CREDENTIALS_REFERENCE_BROWSER'
+        test_var_set 'AMAZON_CLIENT_ID'
```
Comment (MihaiTabara) [inline pushapkscript/docker.d/init_worker.sh:99]: I'm piggybacking on this PR some missing credentials as well to save a deployment later on.

File: pushapkscript/docker.d/worker.yml [hunk 3]
```diff
@@ -119,8 +119,8 @@ products:
               package_names:
                 - "org.mozilla.tv.firefox"
               amazon:
-                client_id: 'dummy'
-                client_secret: 'dummy'
+                client_id: { "$eval": "AMAZON_CLIENT_ID" }
```
Comment (MihaiTabara) [inline pushapkscript/docker.d/worker.yml:122]: Sops is already updated.

File: docker.d/init.sh [hunk 4]
```diff
@@ -79,7 +79,8 @@ export ARTIFACT_UPLOAD_TIMEOUT=1200
 export CONFIG_DIR=/app/configs
 export CONFIG_LOADER=/app/bin/configloader
 export ED25519_PRIVKEY_PATH=$CONFIG_DIR/ed25519_privkey
-export GITHUB_OAUTH_TOKEN=
```
Comment (MihaiTabara) [inline docker.d/init.sh:82]: This was causing https://github.com/mozilla-releng/scriptworker-scripts/blob/master/docker.d/scriptworker.yml#L23 to always be empty.

**Review by catlee — APPROVED** [review]

File: docker.d/scriptworker.yml (hunk 1 above)
Comment (escapewindow) [reply docker.d/scriptworker.yml:31]: This change broke the mac signing pool.

Comment (MihaiTabara) [reply docker.d/scriptworker.yml:31]: @escapewindow Bah, forgot about mac signing pool, sorry about that :(

**Pull request merged**